user/deployment/npm.md

```diff
@@ -132,3 +132,36 @@ after_deploy:
   - ./after_deploy_1.sh
   - ./after_deploy_2.sh
 ```
+
+### Troubleshooting common problems
+
+#### Skipping a deployment with the npm provider because this is not a tagged commit
```

Comment:
What if we just called the section `### Troubleshooting` then? And yes, they are not always problems - they are really just "informational messages" rather than error messages. But if you're trying to get a package published and it says "Skipping blaw blaw" then that's the phrase you end up Googling. That's why I labeled the headers that way, so they would stand out and so Google would find an exact match.

Yes, I know conditional deployment is already covered. But since this is a troubleshooting section, I thought why not be helpful and save the weary Googler from reading the entire manual?